src/test/coverage.go

```diff
@@ -192,6 +193,36 @@ func getStats(coverage core.TestCoverage) stats {
 	return stats
 }
 
+type dirLines struct {
+	covered        int
+	totalCoverable int
+}
+
+func (lines dirLines) getPercentage() float32 {
+	return 100.0 * float32(lines.covered) / float32(lines.totalCoverable)
+}
+
+func getDirectoryCoverage(coverage core.TestCoverage) map[string]float32 {
+	dirCoverage := make(map[string]float32)
+	linesByDir := make(map[string]*dirLines)
+
+	for _, file := range coverage.OrderedFiles() {
```

Comment:
why iterate over OrderedFiles? could you not iterate over just `Files` instead? Can't see how ordering would make a difference...

Reply:
:+1: